src/hffs/fs.py

```diff
@@ -294,33 +333,24 @@ def _fetch_range(self, start, end):
         return r.content
 
     def _initiate_upload(self):
-        self.temp_file = tempfile.NamedTemporaryFile(delete=False)
+        self.temp_file = tempfile.NamedTemporaryFile(dir=huggingface_hub.constants.HUGGINGFACE_HUB_CACHE, delete=False)
```

Comment:
I'm a bit skeptical about putting temporary files in the hub cache folder. This folder is meant to cache files downloaded from the Hub using some versioning (and is "entirely" managed by `hfh`).

If you want to store files in a defined folder specific to `hffs` (instead of `/tmp`), you can use the `assets/` cache from `hfh` ([see docs](https://huggingface.co/docs/huggingface_hub/v0.11.0/en/how-to-cache#caching-assets)). You would have to do something like:
```py
from huggingface_hub import cached_assets_path

# Return a folder path that is guaranteed to exist (lives in `huggingface` cache and will be specific to `hffs`)
assets_path = cached_assets_path(library_name="hffs")
```

Reply:
This is a temp file, so adding a prefix to its name is probably better than having a dedicated directory.

Reply:
Yep, even better. I wasn't sure about the purpose of it but a prefix is definitely better :)